### PR TITLE
LPS-141999 Hide under feature flag

### PR DESCRIPTION
**Pull request by jkappler**

### CODE REVIEW

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/services/LayoutService.js [hunk 1]
```diff
@@ -106,6 +124,17 @@ export default {
 		);
 	},
 
+	/**
+	 * @returns {Promise<{error: Error, friendlyURL: string}>}
+	 */
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/services/LayoutService.js:129]: I think you can remove this :smile:

**Review by p2kmgcl — COMMENTED** [review]
FrontEnd looks good :partying_face:

**Comment by jkappler** [comment]
@lfbesada, conflicts resolved ❤️

**Comment by lfbesada** [comment]
ci:test:relevant

**Review by lfbesada — APPROVED** [review]
Really useful functionality,  works great! Thanks for sharing it!

**Comment by lfbesada** [comment]
ci:forward

**Pull request closed without merging**